lib/src/settings_screen.dart

```diff
@@ -1223,6 +1227,7 @@ class _TextInputSettingsTileState extends State<TextInputSettingsTile> {
           validator: widget.enabled ? widget.validator : null,
           onSaved: widget.enabled ? (value) => _onSave(value, onChanged) : null,
           obscureText: widget.obscureText,
+          keyboardType: widget.keyboardType??TextInputType.text,
```

Comment:
The parameter already has a default value, then why check for null value again?
& again `Is the default value required here?`

Reply:
Removed, as per your suggestion. Thanks.